Script/AtomicEditor/ui/modal/CreateProject.ts

```diff
@@ -127,7 +127,15 @@ class CreateProject extends ModalWindow {
                 this.sendEvent(EditorEvents.LoadProject, { path: folder });
                 return true;
             } else {
-                var message = "Unable to create project for language:  " + selectedLanguage + "\n\nPlease choose a different language";
+                let message = [
+                    "Unable to create project for:",
+                    "",
+                    `language: ${selectedLanguage}`,
+                    `template: ${templateDetail.folder}`,
+                    "",
+                    "Please choose a different language."
+                ].join("\n");
```

Comment:
yeah, I'm still getting into practice using these.  Honestly, that should really be `const message ...` since it's not going to be mutated.  There is also a TSLint rule that can be turned on that bans the use of the `var` statement.  I believe ESLint also has a rule that checks `const` vs `let` usage, but that's not available in TSLint from what I can tell.